src/pages/FilesPage/FilesPage.tsx

```diff
@@ -6,158 +6,82 @@ import Divider from '@mui/material/Divider';
 import Grid from '@mui/material/Grid';
 import LinearProgress from '@mui/material/LinearProgress';
 
-import AddIcon from '@mui/icons-material/Add';
-import DeleteIcon from '@mui/icons-material/Delete';
-import DownloadIcon from '@mui/icons-material/Download';
-
-import {
-	FileMenuButton,
-	FilesContainer,
-	FilesGridContainer,
-	FlexBox,
-	UHeader,
-} from './FilePage.styles';
+import { FilesContainer, FilesGridContainer } from './FilePage.styles';
 
 // helpers
 import { FileItem } from './components/FileItem';
-import { FileLinkResponse, FileService } from 'clients/CoreService';
+import { FileManagerService, ListResponse } from 'clients/CoreService';
 import { useSnackbarOnError } from 'hooks/notification/useSnackbarOnError';
 import { entities } from 'consts/entities';
 import { FileInfo } from './components/FileItem/FileItem';
+import { FolderItem } from './components/FolderItem';
+import { ControlMenu } from './components/ControlMenu';
+import { LocationLinks } from './components/LocationLinks';
 
 function FilesPage() {
-	const coreUrl: string = process.env.REACT_APP_CORE_URL || '';
-	const queryClient = useQueryClient();
+	const [currentLocation, setCurrentLocation] = useState<string>('');
 	const [processingFiles, setProcessingFiles] = useState<Array<FileInfo>>([]);
```

Comment:
We don't use setProcessingFiles anywhere, are you sure that is will be fine?

Reply:
We do not have requests on the backend for this yet, as soon as they appear I will use it, everything is under control)

Reply:
ok)